ocfweb/docs/docs/staff/startertasks.md

```diff
@@ -0,0 +1,89 @@
+[[!meta title="Starter Tasks"]]
+
+Want to dive into learning about technical infrastructure at the OCF, but not
+sure where to start? Here are some self-paced tasks you can do on your own. Feel
+free to ask for help in our [Slack channel](https://ocf.io/slack) or in person
+during [staff hours](https://ocf.io/staffhours)!
+
+Tasks marked with an asterisk (*) require staff privileges. If you want to work
+on these but aren't on staff, let a current staffer know you’re working on
+ starter tasks and we will add you.
+
+These tasks don’t have to be completed in order.
+
+## Connect to the OCF IRC network
+Internet Relay Chat (IRC) is a chat protocol invented in the 80s, an early
+precursor to Slack. The OCF runs an IRC server (since 2001!), which is bridged
```

Comment:
I'm curious how you learned about IRC starting in 2001? Seems like an interesting bit of history.

Reply:
I asked about it in #ocf on Tuesday! Got some fun historical tidbits from dwc and katster as well. Though it was actually March 2002.